src/Uno.UI.RuntimeTests/Tests/HotReload/Frame/HRApp/Tests/RemoteControlExtensions.cs

```diff
@@ -28,6 +29,11 @@ public static async Task UpdateServerFile<T>(string originalText, string replace
 			originalText: originalText,
 			replacementText: replacementText);
 
+		if(message is null)
```

Comment:
Very surprising this PR is green. It should have failed with this formatting error.

```suggestion
		if (message is null)
```

Reply:
This is a separate app, it may not be using the global formatting config.

Reply:
Done!